library/data/recipe/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	moduleName = "recipe"
+	version = "1.0.0"
+	library = "data"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+			api("resource_loader")
+			testmodOnly("qsl_base")
```

Comment:
shouldnt be needed. The `api` dependency is in `testmodOnly`

Reply:
how in hell did it got not pushed